app/views/shared/_consultees_table.html.erb

```diff
@@ -4,7 +4,7 @@
   <tr class="govuk-table__row">
     <th class="govuk-table__header">Reason/constraint</th>
     <th class="govuk-table__header">Consultees</th>
-    <th class="govuk-table__header">Status</th>
+    <th class="govuk-table__header" style="text-align:right">Status</th>
```

Comment:
I think this might be a class you can use
`govuk-!-text-align-right`

https://design-system.service.gov.uk/styles/font-override-classes/

Reply:
Yes! It worked! Thanks for spotting it :)